src/pages/SingleOperatorPage.tsx

```diff
@@ -497,6 +506,141 @@ export const SingleOperatorPage = () => {
+                                                                                    amount: toBN(
+                                                                                        s.amountWei,
+                                                                                    ),
+                                                                                    minimumStakingPeriodSeconds:
+                                                                                        s.minimumStakingPeriodSeconds,
+                                                                                    joinTimestamp:
+                                                                                        s.joinTimestamp,
+                                                                                }),
+                                                                            ),
+                                                                        tokenSymbol:
+                                                                            tokenSymbol,
+                                                                        totalAmount:
+                                                                            element.amount,
+                                                                        onSubmit: async (
+                                                                            sponsorshipId,
+                                                                        ) => {
+                                                                            if (
+                                                                                operatorId
+                                                                            ) {
+                                                                                await forceUnstakeFromSponsorship(
+                                                                                    sponsorshipId,
+                                                                                    operatorId,
+                                                                                )
+
+                                                                                await waitForGraphSync()
+                                                                                refetchQuery(
+                                                                                    operatorQuery,
+                                                                                )
+                                                                            }
+                                                                        },
+                                                                    },
+                                                                )
+                                                            } catch (e) {
```

Comment:
Let's add `if (isRejectionReason(e)) {return}` within this `catch` so that regular modal dismissal is not reported as a failure.